AllASPFuncs.py

```diff
@@ -1221,10 +1244,10 @@ def build_lightcurve_sim(supernova, flux, sigma_flux):
     2.) Soon I will turn many of these inputs into environment variable and they
     should be deleted from function arguments and docstring.
     '''
-    data_dict = {'MJD': np.arange(0, detim, 1), 'true_flux': supernova,
-          'measured_flux':flux , 'flux_error': sigma_flux}
+    data_dict = {'MJD': np.arange(0, detim, 1), 'flux':flux,
```

Comment:
I would prefer defining the MJD for the sims on a different line and then using that variable in the `data_dict` construction.

```
sim_mjd = np.arange(0, detim, 1)
data_dict = {"MJD": sim_mjd, ...
```

Also, insert a space in between `"flux": flux"`